Update namespaces page (#36592)
kubernetes/website: content/en/docs/concepts/workloads/pods/user-namespaces.md

This means containers can run as root and be mapped to a non-root user on the
host. Inside the container the process will think it is running as root (and
therefore tools like `apt`, `yum`, etc. work fine), while in reality the process

> doesn't have privileges on the host. You can verify it from the host by checking the user's 

Reviewer: ```suggestion
doesn't have privileges on the host. You can verify this, for example, if you
```

> container in which the process `ps` is running. The user `ps` shows is not the same as the user 

Reviewer: ```suggestion
check the user that the container process is running as by executing `ps` on the host. The user
```

> you see if the command `id` is execute inside the container. The command `ps` command doesn't 

Reviewer: ```suggestion
in the output of the `ps` command is not the same if you execute the command `id` inside the
```

> show the user ID or user name.

Reviewer 2: Still, this text is confusing. If the `ps` command doesn't show the user ID or the user name, what does "The user `ps` shows ..." mean?

> show the user ID or user name.

Reviewer: ```suggestion
container.
```
